chatwoot/chatwoot
# feat: Rewrite `conversation/labelMixin` to a composable

# Pull Request Template

## Description

This PR will replace the usage of `conversation/labelMixin` with a composable

Fixes https://linear.app/chatwoot/issue/CW-3439/rewrite-conversationlabelmixin-mixin-to-a-composable

## Type of change


## How Has This Been Tested?

**Test cases**

1. Add/remove labels from conversation sidebar
2. See labels are showing up dynamically
3. Check add/remove labels working fine with CMD bar
4. Check card labels in conversation card and SLA reports table.


## Checklist:
